BasicsOOP_lesson5/BasicsOOP_lesson5/BasicsOOP_lesson5/BasicsOOP_lesson5/Task1/RationalNumber.cs

```diff
@@ -0,0 +1,132 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+
+namespace BasicsOOP_lesson5
+{
+    /*     1. Создать класс рациональных чисел. В классе два поля – числитель и знаменатель. 
+     *     Предусмотреть конструктор. Определить операторы ==, != (метод Equals()), <, >, <=, >=, +, –, ++, --. 
+     *     Переопределить метод ToString() для вывода дроби. 
+     *     Определить операторы преобразования типов между типом дробь,float, int. Определить операторы *, /, %.        */
+    class RationalNumber
+    {
+        internal int Numerator { get; set; }
+        internal int Denominator { get; set; }
+
+        RationalNumber() { }
+
+
+        public static bool operator ==(RationalNumber RationalNumber_1, RationalNumber RationalNumber_2) 
+        {
+            if (Equals(RationalNumber_1, RationalNumber_2)) return true;
+            else return false;
```

Comment:
Индусский код detected!
```suggestion
            return Equals(RationalNumber_1, RationalNumber_2);
```